promoted_python_delivery_client/model/blender_rule.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json, LetterCase
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class BlenderRule:
+    attribute_name: str
+    hashed_attribute: int
+    rule: object
```

Comment:
I think I've been seeing "Any" when it can be anything. Is "object" here equivalent or are you communicating it's a specific object, definitely not an int, but you don't have the class type to annotate with?

Reply:
These are basically placeholders for now, but I _believe_ object is most correct for what we're going for in this case, see https://stackoverflow.com/questions/39817081/typing-any-vs-object